core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionPolicyManager.java

```diff
@@ -104,32 +104,32 @@ public TransmissionPolicyManager(boolean throttlingIsEnabled) {
         this.backoffManager = new SenderThreadsBackOffManager(new ExponentialBackOffTimesPolicy());
     }
 
-    /** 
+    /**
      * Suspend the transmission thread according to the current back off policy.
      */
     public void backoff() {
-    	policyState.setCurrentState(TransmissionPolicy.BACKOFF);
-    	long backOffMillis = backoffManager.backOffCurrentSenderThreadValue();
-        if (backOffMillis > 0)
+        policyState.setCurrentState(TransmissionPolicy.BACKOFF);
+        long backOffMillis = backoffManager.backOffCurrentSenderThreadValue();
+        if (backOffMillis > 0) 
         {
             long backOffSeconds = backOffMillis / 1000;
-            InternalLogger.INSTANCE.info("App is throttled, telemetry will be blocked for %s seconds.", backOffSeconds);
+            InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.TRACE, "App is throttled, telemetry will be blocked for %s seconds.", backOffSeconds);
```

Comment:
Will this flood the log and make users unhappy?

Reply:
Well I think it has more consequences than just making user unhappy. Writing logs to disk (i.e file) or console is expensive application and during peak load there can be some sort of throttling in sending some telemetry but because of extensive logging this slow production application down and then can lead to missing important transactions (Say in high throughput e-commerce platform). This should not be log always.

Reply:
I agree with @dhaval24, this shouldn't be `logAlways`. I would go with `debug`.